packages/shared/routes/dashboard/governance/Governance.svelte

```diff
@@ -31,7 +31,7 @@
                 {/each}
             </div>
         </div>
-        <GovernanceDashboard {event} />
+        <GovernanceDashboard {event} account={$selectedAccount} />
```

Comment:
I think `selectedAccount` could be imported directly in `GovernanceDashboard` instead of being a prop since its a global store

Reply:
but this is a mini detail, we can rearrange the props in the polishing phase